mandible/umm/ummg.py

```diff
@@ -0,0 +1,225 @@
+from dataclasses import asdict, dataclass
+from datetime import datetime, timezone
+from pathlib import Path
+from typing import List
+
+# ISO 8601 with Z
+UMM_DATETIME_FORMAT = "%Y-%m-%dT%H:%M:%S.%fZ"
+
+
+def _remove_missing(ummg: dict) -> dict:
+    return {k: v for k, v in ummg.items() if v}
+
+
+@dataclass
+class FileMd(dict):
+    name: str
+    uri: str
+    s3uri: str
+    size: str
+    key: str
+    bucket: str
+
+
+@dataclass
+class CollectionRef:
+    ShortName: str
+    LongName: str
+
+
+@dataclass
+class ProductMd:
+    productCreationDateTime: datetime
+    productStartTime: datetime
+    productStopTime: datetime
+    mission: str
+
+
+@dataclass
+class ProductFilesMd:
+    productFile: FileMd
+    relatedFiles: List[FileMd]
+
+
+@dataclass
+class Meta:
+    CollectionRef: CollectionRef
+    ProductMd: ProductMd
+    ProductFilesMd: ProductFilesMd
+
+
+class UmmgBase():
```

Comment:
Are we putting empty parens or dropping them in class definitions? We should be consistent.